### PR TITLE
Dicom #2239

### PR DESCRIPTION
**Pull request by Jack-Is-2048**

### CODE REVIEW

File: onko_dicom.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# pylint: disable=unnecessary-lambda
```
Comment (pedrooste) [inline onko_dicom.py]: I don't like
Ryan Thoughts?

**Review by pedrooste — DISMISSED** [review]

**Review by ryan-a-git — CHANGES_REQUESTED** [review]
Please remove the .idea directory from the commit.

File: onko_dicom.py (hunk 1 above)
Comment (ryan-a-git) [reply onko_dicom.py]: .idea directory shouldnt be committed at least

Comment (ryan-a-git) [reply onko_dicom.py]: also not sure the purpose of this file

**Comment by Jack-Is-2048** [comment]
> Please remove the .idea directory from the commit.

done

**Review by ryan-a-git — APPROVED** [review]

**Review by pedrooste — APPROVED** [review]

**Pull request merged**